### ISSUE TITLE
Client

### ISSUE THREAD

**Issue by gh-jslee1301** [keybase/client#7513]
['Twas the night before Christmas.]

my log id: d79ed92140abb77b0df84d1c

**Comment by allnash** [comment]
Yeah and then your system was full with a GIANT log file. 1.3 GB in one month. HA!
